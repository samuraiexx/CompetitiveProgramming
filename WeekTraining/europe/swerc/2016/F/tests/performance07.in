10
-1 13893 5619
1 47156 53554
2 52607 32707
1 64044 77540
4 35681 76289
4 35010 28089
5 38994 19395
3 99646 21011
2 22452 30142
9 63991 3893
