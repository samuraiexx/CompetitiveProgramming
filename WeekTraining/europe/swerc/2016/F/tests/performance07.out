0
30142
0
123773
0
0
0
0
0
0
